[IMP] POS: add cards to main page (#17330)
odoo/documentation: content/applications/sales/point_of_sale.rst

:doc:`self-ordering <point_of_sale/extra/self_order>` feature, enabling customers to place orders
and make payments using a dedicated kiosk or their own mobile device.

.. cards::

   .. card:: Use
      :target: point_of_sale/use

>       Set up a POS, manage settings, and handle daily operations.

Reviewer: ```suggestion
      Set up a POS, manage basic settings, and handle daily operations.
```
I added "basic" because I felt "manage settings" is extremely vague. Could be something else if "basic" doesn't really reflect the content :)

   .. card:: Products
      :target: point_of_sale/products

      Configure products, categories, combos, and serial number displays.

   .. card:: Hardware and network
      :target: point_of_sale/hardware_network

      Connect and configure hardware peripherals and network settings.

   .. card:: Shop features
      :target: point_of_sale/shop

      Features specific to retail: manage quotations, "ship later" orders, and barcodes.

   .. card:: Restaurant features
      :target: point_of_sale/restaurant

      Features specific to restaurants: manage floors, tables, tips, and online delivery.

   .. card:: Extra features
      :target: point_of_sale/extra

      Shared features: manage employees, preparation displays, self-ordering, and loyalty programs.

   .. card:: Payment methods
      :target: point_of_sale/payment_methods

>       Configure cash, customer accounts, QR codes, and payment terminals.

Reviewer: ```suggestion
      Configure cash machines, customer accounts, QR codes, and payment terminals.
```
Again, I felt something was missing; having "Configure cash" alone seemed wrong somehow (at least in my mind ;))

Author: then I added cash payments as well, since it's not just cash machines

   .. card:: Reporting
      :target: point_of_sale/reporting

>       View and analyze statistics for orders and sessions.

Reviewer: ```suggestion
   .. card:: Reporting
      :target: point_of_sale/reporting

      View and analyze orders and sessions' statistics.
```
